app/src/main/java/org/torproject/android/ui/onboarding/BridgeWizardActivity.java

```diff
@@ -72,31 +83,33 @@ public void onClick(View v) {
         });
 
         mBtObfs4 = findViewById(R.id.btnBridgesObfs4);
-        mBtObfs4.setOnClickListener(new View.OnClickListener() {
+        mBtObfs4.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
```

Comment:
Changing this listener prevents requests from firing off when the user reselects an already checked RadioButton